docs/Concepts/Privacy/Flexible-PrivacyGroups.md

```diff
@@ -20,23 +20,23 @@ and remove members to and from flexible privacy groups.
 
 !!! important
 
-    Flexible privacy groups are an early access feature. Do not use in production networks.
+    Flexible privacy groups are an early access feature. Don't use in production networks.
 
-    The flexible privacy group interfaces might change between releases. There might not be an
+    The flexible privacy group interfaces might change between releases. There may not be an
```

Comment:
I agree "may" sounds better. The [style guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may) does say to use "might" for expressing probability instead of "may" though.

Reply:
Good catch. I'll actually change back to "might" since it's used in the previous sentence.